captcha/fields.py

```diff
@@ -107,6 +107,9 @@ def validate(self, value):
             # value back.
             score = float(check_captcha.extra_data.get("score", 0))
 
+            # Update widget attrs with returned_score
+            self.widget.attrs["returned_score"] = score
```

Comment:
I feel like setting it on the widget doesn't make sense; why not put this in `cleaned_data` or is it already populated?